sklearn/metrics/_pairwise_distances_reduction/_argkmin.pyx.tp

```diff
@@ -4,7 +4,7 @@ from cython cimport final
 from cython.parallel cimport parallel, prange
 
 from ...utils._heap cimport heap_push
-from ...utils._sorting cimport simultaneous_sort
+from ...utils._sorting cimport simultaneous_quicksort as simultaneous_sort
```

Comment:
What's the reason for using an alias ? Since there are only a few occurences I think it's fine to directly replace by the new name (here and in other files)